lib/resources/dstu2/document_reference_create.yaml

```diff
@@ -0,0 +1,130 @@
+---
+field_name_base_url: http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference
+fields:
+- name: resourceType
```

Comment:
A couple things going on here:

1. The resourceType link was basically working as intended to take you to the top level of the DocumentReference field list, but it should be linking to the resourceType definition instead, so I have added the url for the field to point to the correct location.

2. The url generation logic was incorrectly flipped during some refactoring so that the override urls would only be used if there was no base url defined. I rearranged that, so now the overrides should link properly for the relevant fields (resourceType, content.attachment.contentType, content.attachment.data)